### ISSUE TITLE
Tabs are not accessible

### ISSUE THREAD

**Issue by siddharthkp** [auth0/cosmos#923]
**Describe the bug**
The Tabs cannot be navigated using the keyboard by hitting tab. (the irony!)

**Expected behavior**
You should be able to move through Tab Links by using tab key, and selecting one by using the enter key

**Screenshots**
![tab-tab](https://user-images.githubusercontent.com/1863771/45710841-2fea3c00-bb5e-11e8-9d61-6a63df033b96.gif)

**Cosmos version:**
0.5.2

**Additional context**
Could be a case of non-semantic HTML output

**Comment by andresgalante** [comment]
Since you are at it, instead of just adding tab indexes, why don't we make the component fully accesible.

The pattern for tab panel is well defined. I can probably help making the HTML and CSS changes when I have time if someone can point me to the right direction to know how to add attributes to an HTML element and takes care of the keyboard interaction.

https://www.w3.org/TR/wai-aria-practices-1.1/#tabpanel

**Comment by andresgalante** [comment]
Even though there was a PR that made tabs better they were not following ARIA standards.

On #986 we are improving tab accessibility, but there is a bit more work to do.

@francocorreasosa and I will take a look at it and implement a solid solution

**Comment by andresgalante** [comment]
while making them accessible, make sure they behave and look correctly within the dialog
